app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -64,9 +64,9 @@ def dest_folder
 
     host_dc = dest_host.parent_datacenter || dest_host.ems_cluster.parent_datacenter
 
-    # Pick 'Discovered virtual machine' or its parent folder in the destination datacenter
+    # Pick the parent folder in the destination datacenter
     vm_folder = "#{host_dc.folder_path}/vm"
-    find_folder("#{vm_folder}/Discovered virtual machine", host_dc) || find_folder(vm_folder, host_dc)
+    find_folder(vm_folder, host_dc)
```

Comment:
Yes - I only left it there for readability.  I'll pull it out though as the `find_folder` method will still be easy to read since the string referenced by `vm_folder` is much shorter than it originally was.